language/tools/vm-genesis/src/genesis_gas_schedule.rs

```diff
@@ -139,22 +136,16 @@ static INITIAL_GAS_SCHEDULE: Lazy<Vec<u8>> = Lazy::new(|| {
         ),
         (Nop, GasCost::new(10, 1)),
     ];
+    instrs.sort_by_key(|cost| instruction_key(&cost.0));
```

Comment:
Might add a comment here saying why we need to sort to prevent future gotchas on this